simtools/production_configuration/interpolation_handler.py

```diff
@@ -0,0 +1,197 @@
+"""Interpolates between instances of StatisticalErrorEvaluator using EventScaler."""
+
+import astropy.units as u
+import matplotlib.pyplot as plt
```

Comment:
I do suggest to separate plotting functions into their own module.

importing matplotlib is slow and it is a large package. Not ideal to import it each time we want to run interpolation.

Reply:
Moved the import to the plotting function. Let me know if this is also fine.

Reply:
That is only the 2nd best solution, as it requires to disable a pylint rule (and we should minimize disabling them). Fine for now, but should fix this in future (throughout the code).